Sources/Data/models/Model.xcdatamodeld/Model16.xcdatamodel/contents

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Recreate this model to pick up database changes from:
1. https://github.com/brave/brave-ios/commit/5fa92d32436e828beb0ee60c6a9c1e07e8dadf29
2. https://github.com/brave/brave-ios/commit/daf4b67868a9a76644ffc22552c24ca03bfbd905

Reply:
Ok will do that